### ISSUE TITLE
Make Screen reader instructions match the visual notifications for audio player

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1850]
## Problem

The screen reader feedback does not match the visual notification. Visual notification communicates the left and right arrows seek through the track, and screen reader notification only mentions the use of the Spacebar.

## Description

All instructions should be constructed to be more DRY, to ensure that all users get the same amount of feedback.

## Additional context

Reported by Joe Dolson (https://make.wordpress.org/openverse/2022/09/07/frontend-release-v3-4-8-and-a-call-for-a11y-testing/#comment-95)

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.
